src/style.css

```diff
@@ -0,0 +1,64 @@
+.form-field {
+  width: 100%;
+}
+.bigSearch {
+  height: 70px;
+  width: 900px;
+  left: 0px;
+  top: 0px;
+  border-radius: 4px;
+  padding-left: 50px;
+  font-weight: 300;
+  display: block;
+  margin-top: 20px;
+  border: 2px solid lightgray;
+}
+.pic1 {
+  height: 19px;
+  width: 19px;
+  left: 26px;
+  top: 48px;
+  border-radius: 0px;
+  box-sizing: border-box;
+  position: absolute;
+}
```

Comment:
allways add 1 empty line between styles of elements:
```suggestion
.form-field {
  width: 100%;
}

.bigSearch {
  height: 70px;
  width: 900px;
  left: 0px;
  top: 0px;
  border-radius: 4px;
  padding-left: 50px;
  font-weight: 300;
  display: block;
  margin-top: 20px;
  border: 2px solid lightgray;
}

.pic1 {
  height: 19px;
  width: 19px;
  left: 26px;
  top: 48px;
  border-radius: 0px;
  box-sizing: border-box;
  position: absolute;
}
```

Reply:
![image](https://user-images.githubusercontent.com/109065303/179044523-c43ebf69-8d69-434f-bbde-b90e0645311f.png)
I do have empty lines there. Could it be an issue with Github? Kinda like the one I had with spaces in the previous task?